src/lib-components/forms/DateRangePicker.vue

```diff
@@ -51,7 +53,34 @@ onMounted(() => {
         })
       }
     },
-  })
+  }
+
+  if (props.allowedRangeInDays) {
+    // Set the range to a prefilled value given the allowed range
+    const daysAgo = new Date()
+    opts.defaultDate = [
```

Comment:
Watch out for this one.  Looks like you'll end up with what appears to be a completed field, but a `modelValue` that isn't updated, since it's only updated through flatpicker's `onClose` event.